### ISSUE TITLE
Node 0.12

### ISSUE THREAD

**Issue by kbariotis** [davidmarkclements/cute-stack#9]
Looks like can't be installed on 0.12

```
npm install cute-stack --save
-
> v8-flags@0.1.2 preinstall /Users/Bariotis/www/administration-dashboard/node_modules/cute-stack/node_modules/v8-flags-object-proxy/node_modules/v8-flags
> node ./scripts/preinstall

/
> v8-flags@0.1.2 install /Users/Bariotis/www/administration-dashboard/node_modules/cute-stack/node_modules/v8-flags-object-proxy/node_modules/v8-flags
> node-gyp rebuild

child_process: customFds option is deprecated, use stdio instead.
  CXX(target) Release/obj.target/v8_flag_list/src/v8_flag_list.o
../src/v8_flag_list.cc:4:10: fatal error: 'atomicops.h' file not found
#include "atomicops.h"
         ^
1 error generated.
make: *** [Release/obj.target/v8_flag_list/src/v8_flag_list.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/Bariotis/.nvm/versions/node/v0.12.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:267:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1067:12)
gyp ERR! System Darwin 14.1.0
gyp ERR! command "node" "/Users/Bariotis/.nvm/versions/node/v0.12.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/Bariotis/www/administration-dashboard/node_modules/cute-stack/node_modules/v8-flags-object-proxy/node_modules/v8-flags
gyp ERR! node -v v0.12.0
gyp ERR! node-gyp -v v1.0.2
gyp ERR! not ok
npm ERR! Darwin 14.1.0
npm ERR! argv "/Users/Bariotis/.nvm/versions/node/v0.12.0/bin/node" "/Users/Bariotis/.nvm/versions/node/v0.12.0/bin/npm" "install" "cute-stack" "--save"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! v8-flags@0.1.2 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the v8-flags@0.1.2 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the v8-flags package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls v8-flags
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Bariotis/www/administration-dashboard/npm-debug.log
```

**Comment by davidmarkclements** [comment]
ah... that'll be the v8 flags module... 

hmm. if you're not looking for CLI functionality, installing 1.2.x should work while we figure that one out

```
npm install --save cute-stack@1.2
```

**Comment by kbariotis** [comment]
Great. Thank you. :)

**Comment by marcooliveira** [comment]
:+1:

**Comment by davidmarkclements** [comment]
I've put in a partial PR to the v8-flags model to get the ball rolling

https://github.com/thlorenz/v8-flags/pull/4

If anyone has more knowledge on C++ (and/or v8) please feel free to chip in!

**Comment by ORESoftware** [comment]
:+1:

**Comment by ORESoftware** [comment]
`+1`

**Comment by davidmarkclements** [comment]
@kbariotis @marcooliveira @ORESoftware  - this is now fixed in 1.4
